README.md

```diff
@@ -15,6 +15,11 @@ case you want to stream something, maybe used with a second account. For
 anything else, this client has way too many things that work less well than in
 the original client.
 
+## DISCLAIMER!!!
+
+  ***USING THIRD PARTY CLIENTS IS AGAINST <a href="https://discord.com/terms">DISCORD'S TERMS OF SERVICE</a>, WE DO NOT TAKE ANY RESPONSABILITY FOR ANYTHING CAUSED DUE TO THE VIOLATION OF DISCORD'S TOS***
+  
+  
```

Comment:
Thanks. I wouldn't make the notice this scary, but having one is definitely good.

```suggestion
Technically this could be against
[Discord's TOS](https://discord.com/terms#software-in-discord%E2%80%99s-services),
so be warned. Discord probably won't ban you for using this, but if they do I
told you and it's not my fault.

```

Reply:
haha, i like the laid-back tone of that :)

fully up to you, it's your project

Reply:
Yeah the serious disclaimer is basically covered by the license https://github.com/maltejur/discord-screenaudio/blob/master/LICENSE#L600